packages/experimental/Expander/README.md

```diff
@@ -0,0 +1,452 @@
+
+## Usage
+
+Import Expander from @fluentui-react-native/experimental-expander
+
+`import { Expander } from @fluentui-react-native/experimental-expander`
+
+Add `Expander` as follows:
+
+```javascript
+<Expander collapsedHeight={50} expandedHeight={100}>
+  <Text>Text in the header</Text>
+  <Text>Text in the content</Text>
+</Expander>
+```
+
+The first child element of Expander will be assigned to the header, and the second child element will be assigned to the content. If you would like multiple components in the header or the content, wrap them in `View` as follows:
+
+```javascript
+<Expander collapsedHeight={75} expandedHeight={150}>
+  <View>
+    <Text>Text in the header</Text>
+    <Text>Second line of text in the header<Text>
+  </View>
+  <View>
+    <Text>Text in the content</Text>
+    <Text>Second line of text in the content<Text>
+  </View>
+</Expander>
+```
+
+### Props
+
+- [`expandDirection`](#expanddirection)
+- [`expanded`](#expanded)
+- [`enabled`](#enabled)
+- [`height`](#height)
+- [`expandedHeight`](#expandedheight)
+- [`collapsedHeight`](#collapsedheight)
+- [`onCollapsing`](#oncollapsing)
+- [`onExpanding`](#onexpanding)
+- [`onChange`](#onchange)
+
+### Tokens
+
+- [`width`](#width)
+- [`contentHorizontalAlignment`](#contenthorizontalalignment)
+- [`contentVerticalAlignment`](#contentverticalalignment)
+- [`headerBackground`](#headerbackground)
+- [`headerForeground`](#headerforeground)
+- [`headerForegroundPointerOver`](#headerforegroundpointerover)
+- [`headerForegroundPressed`](#headerforegroundpressed)
+- [`headerBorderBrush`](#headerborderbrush)
+- [`headerBorderPointerOverBrush`](#headerborderpointeroverbrush)
+- [`headerBorderPressedBrush`](#headerborderpressedbrush)
+- [`headerDisabledForeground`](#headerdisabledforeground)
+- [`headerDisabledBorderBrush`](#headerdisabledborderbrush)
+- [`headerBorderThickness`](#headerborderthickness)
+- [`contentBackground`](#contentbackground)
+- [`contentBorderBrush`](#contentborderbrush)
+- [`chevronBackground`](#chevronbackground)
+- [`chevronForeground`](#chevronforeground)
+- [`chevronPointerOverBackground`](#chevronpointeroverbackground)
+- [`chevronPointerOverForeground`](#chevronpointeroverforeground)
+- [`chevronPressedBackground`](#chevronpressedbackground)
+- [`chevronPressedForeground`](#chevronpressedforeground)
+- [`chevronBorderThickness`](#chevronborderthickness)
+- [`chevronBorderBrush`](#chevronborderbrush)
+- [`chevronBorderPointerOverBrush`](#chevronborderpointeroverbrush)
+- [`chevronBorderPressedBrush`](#chevronborderpressedbrush)
+
+### Important notes
+
+- `collapsedHeight` and `expandedHeight` must be set for Expander to display correctly
+- Do not set the `height` or `onChange` props. These props are used to update the height of Expander, and the Expander height will not update if these props are overridden. If you need to override either of these props, you must implement the height change functionality as well.
```

Comment:
What would the user need to do to override the onChange?

Reply:
I meant if the user implements an onChange callback, our callback that adjusts the height will be gone. If the user wants to use onChange, they should use the onExpanding and onCollapsing events instead.

Reply:
Ok, I'd then rephrase this part: "you must implement the height change functionality as well"